lib/HunkView.ml

```diff
@@ -2,10 +2,14 @@ open Nottui
 module W = Nottui_widgets
 
 (* Types *)
-
 type line = Change of string | Common of string | Empty
+type rendering_mode = Color | TextMarkers
 
 (* Utility Functions *)
+let line_marker (sign : string option) (content : Ui.t) : Ui.t =
+  match sign with
```

Comment:
This function is used only in one place, where `sign` is never `None`. Did you mean to also use it in other places ?

Reply:
it was previously used in another place but I reverted that code in between my commits, so it is only used in one place now. Need to fix that